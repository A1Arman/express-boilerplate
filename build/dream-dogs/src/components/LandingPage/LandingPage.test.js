import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from './LandingPage';

describe(`LandingPage component`, () => {
    it('renders without crashing', () => {
        const post = [
            {
                dog_name: 'Jax',
                email: 'armanbhimani99@gmail.com',
                breed: 'Maltipoo',
                birthdate: '2018-06-18',
                lifestyle: 'Small dog that loves to run around. Will fit well in smaller homes as well as larger homes. Will need atleast an hour of exercise to keep out of trouble.',
                owner_id: 1
            },
            {
                dog_name: 'Nova',
                email: 'armanbhimani99@gmail.com',
                breed: 'Maltipoo',
                birthdate: '2018-06-18',
                lifestyle: 'Small dog that loves to run around. Will fit well in smaller homes as well as larger homes. Will need atleast an hour of exercise to keep out of trouble.',
                owner_id: 2
            },
            {
                dog_name: 'Rex',
                email: 'armanbhimani99@gmail.com',
                breed: 'Golden Retriever',
                birthdate: '2018-06-18',
                lifestyle: 'Small dog that loves to run around. Will fit well in smaller homes as well as larger homes. Will need atleast an hour of exercise to keep out of trouble.',
                owner_id: 1
            },
        ]
        const wrapper = shallow(<LandingPage posts={post}/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});