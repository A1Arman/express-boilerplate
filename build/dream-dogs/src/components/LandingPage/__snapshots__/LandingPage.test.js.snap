// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders without crashing 1`] = `
<Fragment>
  <main
    className="landing-main"
    role="main"
  >
    <header
      className="landing-header"
      role="banner"
    >
      <h1
        id="landing-title"
      >
        Dream Dogs
      </h1>
      <h2
        id="phrase"
      >
        Find Your Dream Dog.
      </h2>
      <Link
        to="/posts"
      >
        <button
          className="btn"
        >
          Demo App
        </button>
      </Link>
    </header>
    <section
      className="desc-section"
    >
      <header>
        <h3>
          What is Dream Dogs?
        </h3>
      </header>
      <p
        className="desc"
      >
        Dream Dogs helps owners struggling to provide for their dogs to find a safe new home. Owners can create posts and also include a description of the dog, and the lifestyle needs of the dog.
      </p>
    </section>
    <section>
      <h3>
        Dream Dogs Credentials
      </h3>
      <p>
        Email: armanibhimani@gmail.com
      </p>
      <p>
        Password: password
      </p>
    </section>
    <Spotlight />
  </main>
</Fragment>
`;
